go-controller/pkg/ovn/gateway_init.go

```diff
@@ -140,6 +141,16 @@ func gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet, hostSubnets []*n
 			kapi.ProtocolUDP:  k8sNSLbUDP,
 			kapi.ProtocolSCTP: k8sNSLbSCTP,
 		}
+		workerK8sNSLbTCP, workerK8sNSLbUDP, workerK8sNSLbSCTP, err := loadbalancer.GetWorkerLoadBalancers(nodeName)
```

Comment:
I think this should not be conditioned to `if l3GatewayConfig.NodePortEnable {`

https://github.com/ovn-org/ovn-kubernetes/pull/2055/files#diff-9c68f154bc9189eb61d17c17d1392ee36c13a497f4f742a98250c2e0a272d48cR131

anymore, as these loadbalancers are not used only for nodeports

Reply:
i agree. will fix.